app/src/main/res/layout/activity_main.xml

```diff
@@ -5,20 +5,29 @@
     android:focusableInTouchMode="true"
     tools:context=".MainActivity">
 
-    <com.mapzen.tangram.MapView
-        android:id="@+id/map"
+    <include layout="@layout/splash_screen"/>
+
+    <RelativeLayout
```

Comment:
Rather than moving the rest of the views into a nested RelativeLayout, it should be possible to declare the splash screen layout at the bottom of this file.

This way it will be drawn on top of everything else. There will be no need to hide/show the map programatically and it will keep the layout file a little bit cleaner.